src/tablib/formats/_xlsx.py

```diff
@@ -57,15 +57,23 @@ def export_set(cls, dataset, freeze_panes=True, invalid_char_subst="-"):
         return stream.getvalue()
 
     @classmethod
-    def export_book(cls, databook, freeze_panes=True):
-        """Returns XLSX representation of DataBook."""
+    def export_book(cls, databook, freeze_panes=True, invalid_char_subst="-"):
+        """Returns XLSX representation of DataBook.
+
+        If dataset.title contains characters which are considered invalid for an XLSX file
+        sheet name (http://www.excelcodex.com/2012/06/worksheets-naming-conventions/), it will
```

Comment:
plural, to match "characters"

```suggestion
        sheet name (http://www.excelcodex.com/2012/06/worksheets-naming-conventions/), they will
```

Reply:
English is not my strong side :(